tests/core/io/test_http_client.h

```diff
@@ -90,6 +92,7 @@ TEST_CASE("[HTTPClient] verify_headers") {
 	ERR_PRINT_ON;
 }
 
+#if defined(MODULE_MBEDTLS_ENABLED) || defined(WEB_ENABLED)
```

Comment:
I don't expect tests to be run on web (I don't even know *how* to do that) but the dedicated test is because web has its own http client, so just made this change for completeness